xml_converter/generators/code_generator.py

```diff
@@ -359,13 +388,55 @@ def build_example(self, type: str, applies_to: List[str], xml_field: str, exampl
 
         return example
 
+    ############################################################################
+    # Generate Node Types
+    #
+    # This will output code for a single category of nodes.
+    ############################################################################
+    def generate_node_types(self, metadata: Any, attributenames: Dict[str,str],page: str) -> Tuple[List[str],List[str]]:
+        
+        field_rows: List[str] = []
+        includelist: Set[str] = set()
+        typechange: Dict[str,str] = {
```

Comment:
Lets be slightly more descriptive with this name and call it something like `documentation_type_to_cpp_type` or `doc_type_to_cpp_type`.

Reply:
done. doc_type_to_cpp_type